scripts/irods/test/test_imeta_error_handling.py

```diff
@@ -86,7 +86,7 @@ def test_imeta_add_return_code(self):
         # In unixODBC versions > 2.3.5, the database plugin does not return CATALOG_ALREADY_HAS_ITEM_BY_THAT_NAME
         # but instead it returns CAT_SQL_ERR. We support platforms that use both 2.3.4 and 2.3.6, so either error
         # code is acceptable for this test.
-        self.assertTrue('CATALOG_ALREADY_HAS_ITEM_BY_THAT_NAME' in err or 'CAT_SQL_ERR' in err)
+        self.assertTrue('CATALOG_ALREADY_HAS_ITEM_BY_THAT_NAME' in err)
```

Comment:
Prefer the more targeted versions of the assert functions over `assertTrue` and `assertFalse` as they lead to clearer failure messages when they occur.

`assertTrue` -> `assertIn`